openml/datasets/dataset.py

```diff
@@ -992,6 +969,80 @@ def _to_dict(self) -> "OrderedDict[str, OrderedDict]":
 
         return data_container
 
+    def _get_arff(
```

Comment:
Is there a reason you moved this function? This makes it hard to review, and I would appreciate if you could move it back to its original place.